en/db-migrations.md

```diff
@@ -119,6 +119,33 @@ $migration::run([
 ]);
 ```
 
+## Migration methods
+
+Each migration is an separate class that works as entity for specific database table.
+Inside each class there are different methods that can occur during migration running.
+
+You can also execute whatever logic you want inside each migration, without any limitation.
```

Comment:
Each migration file (and class) can implement specific methods, that will be executed based on the operation requested. There are no restrictions on the logic encapsulated in each method.